tests/Form/SelectTest.php

```diff
@@ -13,6 +13,12 @@ public function testMultiselect()
         $page = $this->getSession()->getPage();
         $this->assertEquals('Multiselect Test', $webAssert->elementExists('css', 'h1')->getText());
 
+        $selectWithoutOption = $webAssert->fieldExists('select_without_option');
+        $this->assertNull($selectWithoutOption->getValue());
+
+        $selectWithNoOptionSelected = $webAssert->fieldExists('select_first_option_is_selected_by_default');
```

Comment:
Is this a default behavior for all the drivers to return 1st option value in select if none are selected?

Reply:
For a select not multiple, it's the default behavior in all browsers. So should be the default for our drivers as well. If that's not the case, it's another thing to fix.